src/psyclone/psyir/nodes/__init__.py

```diff
@@ -60,6 +61,7 @@
         'UnaryOperation',
         'BinaryOperation',
         'NaryOperation',
+        'Range',
```

Comment:
My pylint complains about indentation (for all entries). If this is not the case for you then please ignore.

Reply:
My pylint is very happy!

Reply:
OK.